config/spryk/spryks/Spryker/Glue/AddGlueMapperFactoryMethod.yml

```diff
@@ -0,0 +1,35 @@
+spryk: method
+description: "Adds a create mapper method to the *module*Factory."
+mode: both
+
+arguments:
```

Comment:
when creating a mapper factory method, namespace output that is returned from the method is wrong

Reply:
I'm not sure, looks ok to me:

```/**
     * @return \Spryker\Glue\Agent\Processor\Mapper\AgentMapperInterface
     */
    public function createAgentMapper(): \Spryker\Glue\Agent\Processor\Mapper\AgentMapperInterface
    {
        return new \Spryker\Glue\Agent\Processor\Mapper\AgentMapper();
    }```